src/components/UserSelector.tsx

```diff
@@ -1,19 +1,40 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { User } from '../types/User';
+
+type Props = {
+  users: User[];
+  onSelect: (user: User) => void;
+  selectedUser: User | null;
+  fetchPosts: (userId: number) => void;
+  usersIsLoaded: boolean;
+};
+
+export const UserSelector: React.FC<Props> = (
+  {
+    users, onSelect, selectedUser, fetchPosts, usersIsLoaded,
+  },
```

Comment:
```suggestion
  {
    users,
    onSelect,
    selectedUser,
    fetchPosts,
    usersIsLoaded,
  },
```

Reply:
The better variant of formatting in this case :arrow_up: